plugins/synced_folders/rsync/helper.rb

```diff
@@ -63,7 +63,11 @@ def self.rsync_single(machine, ssh_info, opts)
         opts[:owner] ||= ssh_info[:username]
         opts[:group] ||= ssh_info[:username]
 
+        # set log level
+        log_level = ssh_info[:log_level] || "FATAL"
+
```

Comment:
I don't recall at the moment, but this was either copied from older code or prevented errors from log_level not being set in my testing way back when.